src/thinkspace/api/thinkspace-readiness-assurance/db/associations.yml

```diff
@@ -4,7 +4,7 @@
     - {belongs_to: thinkspace/common/user, readonly: true}
     - {has_many: thinkspace/readiness_assurance/responses, dependent: destroy}
   serializers:
-    - attributes: [id, title, authable_type, authable_id, question_settings, ra_type]
+    - attributes: [id, title, authable_type, authable_id, question_settings, questions, settings, ra_type, answers]
```

Comment:
This was something I think we talked about at some point, and serialized to get the answer functionality working in the builder without too much hassle.

As far as what you mention in `irats_controller`, should we be using the pairing of `irat#assessment` and `trat#assessment` to get the assessments in question? Possibly using the `assignment#phase_componentables` call to get the relevant assessment ids, then loading them again using the `irat` and `trat` specific calls?